### ISSUE TITLE
What Ruby libraries should we support?

### ISSUE THREAD

**Issue by TylerFisher** [news-catalyst/presspass-frontend#45]

**Comment by jacqui** [comment]
Devise is the most popular, I think we should start there.

I spent a fair amount of time today getting my laptop setup to work with ruby and rails - it's been awhile! Wow this all takes a long time to install... but I'm there now. I started working on a new sample app for the login functionality in rails, should have something tomorrow to look at ✌️

**Comment by jacqui** [comment]
Whoops, I didn't mean to close the issue :)

**Comment by jacqui** [comment]
I have a working rails integration with PressPass auth, hip hip! It ended up being pretty easy; I blame my pandemic-brain for stumbling with it when I first started on it last week. 

We have a sample client app repo at https://github.com/news-catalyst/presspass-rails with instructions in the README :)  I did run into some trouble with all the OIDC ruby gems I tried mandating `https` for the auth provider, and while I did manage to get the dev squarelet up and running on `https` it was a pain. And I figured out a ruby workaround until we're fully hosted that works just fine.
